src/utils/getComputedOcpReportItems.ts

```diff
@@ -98,12 +99,13 @@ export function getUnsortedComputedOcpReportItems({
             : '';
         const id = `${value[idKey]}${idSuffix}`;
         let label;
-        if (labelKey === 'cluster' && cluster_alias) {
+        const itemLabelKey = getItemLabel({ report, labelKey, value });
+        if (itemLabelKey === 'cluster' && cluster_alias) {
           label = cluster_alias;
-        } else if (value[labelKey] instanceof Object) {
-          label = (value[labelKey] as OcpDatum).value;
+        } else if (value[itemLabelKey] instanceof Object) {
+          label = (value[itemLabelKey] as OcpDatum).value;
         } else {
-          label = value[labelKey];
+          label = value[itemLabelKey];
```

Comment:
I'm wondering if this is old, unused code because the debugger never seems to step into the `else if` statements?

It appears that the itemLabelKey could return something with "tag:" prefix. However, will the report "values" ever contain "tag:project", for example?

![Screen Shot 2020-02-06 at 9 17 38 AM](https://user-images.githubusercontent.com/17481322/73945160-e10f8480-48c1-11ea-8a84-fa5436cdb867.png)

Reply:
As mentioned here: https://github.com/project-koku/koku/pull/1690#issuecomment-582891216
This is special case logic to colliding keywords. The `tag:` prefix wouldn't be placed on projects, but it will now for a tag like `cost` -> `tag:cost`, so it doesn't collide with the total `cost` value.

If you reference the API output shown in the following link you can see the difference: https://github.com/project-koku/koku/pull/1690#issue-371042654

Hope that clarifies.

Reply:
Ok, I think I got it. You're adding the tag prefix if and only if there is a collision.

I see that collision (and prefix) with Azure after group_by[tag:cost] now.

Reply:
yep.